packages/twenty-front/src/modules/activities/emails/components/EmailThreadMessage.tsx

```diff
@@ -3,9 +3,12 @@ import styled from '@emotion/styled';
 
 import { EmailThreadMessageBody } from '@/activities/emails/components/EmailThreadMessageBody';
 import { EmailThreadMessageBodyPreview } from '@/activities/emails/components/EmailThreadMessageBodyPreview';
+import { EmailThreadMessageReceivers } from '@/activities/emails/components/EmailThreadMessageReceivers';
 import { EmailThreadMessageSender } from '@/activities/emails/components/EmailThreadMessageSender';
 import { EmailThreadMessageParticipant } from '@/activities/emails/types/EmailThreadMessageParticipant';
 
+const PARTICIPANT_FROM_ROLE = 'from';
```

Comment:
I think from here is confusing. Can we introduce a ParticipantRole type and use `ParticipantRole.From` instead?

Reply:
updated!